lib/clients/backbeat-2017-07-01.api.json

```diff
@@ -897,13 +944,34 @@
         "BatchDelete": {
             "http": {
                 "method": "POST",
-                "requestUri": "/_/backbeat/batchdelete"
+                "requestUri": "/_/backbeat/batchdelete/{Bucket}/{Key+}"
```

Comment:
Are those new params required? If so, did we check that transient source complies with the change (the only other user of batchdelete route for now)?

Reply:
It is my understanding that without explicitly stating them as required below, then they can be omitted. I.e. currently this route has no required params:

```
...
"required": [
],
...
```

Will validate with a transient source.

Reply:
ok, so the API wrapper would automatically set the URL to `/_/backbeat/batchdelete` if none of Bucket or Key are provided? I don't know, but if it's the case, then we should be good.

Zenko E2E have a transient source test, it should catch any critical issue there.

Reply:
It seems to send placeholders for the bucket and key `/_/backbeat/batchdelete/{Bucket}/{Key+}`. After testing E2E and transient source it doesn't appear to affect behavior of the route.